src/state/persisted/schema.ts

```diff
@@ -19,12 +22,26 @@ const accountSchema = z.object({
 })
 export type PersistedAccount = z.infer<typeof accountSchema>
 
+/**
+ * The current account. Stored in the `currentAccount` field.
+ *
+ * In previous versions, this included tokens and other info. Now, it's used
+ * only to reference the `did` field, and all other fields are marked as
+ * optional. They should be considered deprecated and not used, but are kept
+ * here for backwards compat.
+ */
+const currentAccountSchema = accountSchema.extend({
+  service: z.string().optional(),
+  handle: z.string().optional(),
```

Comment:
This relaxes the type a little bit so that `did` is the only required field. We should be careful because this means the app can write a less "complete" version of `currentAccount`, and if ever roll this back, the old version of the code would not be able to work with that (and would also believe falsehoods about it due to wrong types where these fields were required).

In this PR, there are no changes to how `currentAccount` is written though — so that should be fine.